src/main/java/com/vinylteam/vinyl/entity/Shop.java

```diff
@@ -51,6 +52,14 @@ public void setName(String name) {
         this.name = name;
     }
 
+    public String getSmallImageLink() {
+        return smallImageLink;
+    }
+
+    public void setSmallImageLink(String smallImageLink) {
+        this.smallImageLink = smallImageLink;
+    }
+
     @Override
     public boolean equals(Object o) {
```

Comment:
Add smallImageLink to hashCode and equals.

Reply:
and to toString()  actually, thanks